File: Java/Basics To Java/src/F1/F1_3.java
package F1;

import java.util.Scanner;

public class F1_3 {
	public static void mainx(String[] args) {
		Scanner sc = new Scanner(System.in);
		char znak;
		int pocet, presun, sub;

		System.out.println("zadej znak a bude zakódován");
		znak = sc.next().charAt(0);
		System.out.println("zadej o kolik se posune");
		pocet = sc.nextInt();
		presun = kodovaniF(znak, pocet); // funkce
		System.out.printf("pùvodní znak: %c %nposunutí: %d %nzakódování: %c", znak, pocet,presun); // fce
	}

	public static int kodovaniF(char klavesa, int cislo) // funkce
	{
		int vystup;
		vystup = klavesa + cislo;
		if (vystup > 122)
			vystup = vystup - 26;
		return vystup;
	}
}

File: C#/Basic logic/F1/F1_3.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace F1
{
    internal class F1_3
    {
        static void Mainx(string[] args)
        {
            char znak;
            int pocet, presun, sub;

            Console.WriteLine("zadej znak a bude zakódován");
            znak = char.Parse(Console.ReadLine());
            Console.WriteLine("zadej o kolik se posune");
            pocet = int.Parse(Console.ReadLine());
            presun = kodovaniF(znak, pocet); //funkce
            kodovaniS(znak, pocet, out sub); //subrutina
            Console.WriteLine("původní znak: {0} \nposunutí: {1} \nzakódování: {2}", znak, pocet, (char)presun); //fce
            Console.WriteLine("původní znak: {0} \nposunutí: {1} \nzakódování: {2}", znak, pocet, (char)sub); //sbrtn
        }
        static int kodovaniF(char klavesa, int cislo) //funkce
        {
            int vystup;
            vystup = klavesa + cislo;
            if (vystup > 122)
                vystup = vystup - 26;
            return vystup;
        }

        static void kodovaniS(char klavesa, int cislo, out int s) //subrutina
        {
            int vystup;
            vystup = klavesa + cislo;
            if (vystup > 122)
                vystup = vystup - 26;
            s = vystup;
        }
    }
}
